dll/win32/shell32/res/rgs/usernotification.rgs

```diff
@@ -0,0 +1,43 @@
+HKCR
+{
+	NoRemove AppID
+	{
+		{0010890E-8789-413C-ADBC-48F5B511B3AF}
+		{
+			val DllSurrogate = s ''
+			val RunAs = s 'Interactive User'
+		}
+	}
+	NoRemove CLSID
+	{
+		ForceRemove {0010890E-8789-413C-ADBC-48F5B511B3AF} = s 'User Notification'
+		{
+			InprocServer32 = s '%MODULE%'
```

Comment:
The COM object runs in-process, what happens if you close the application adding it before the balloon is done showing?
Does the tray disappear or stay?

Reply:
The tray stays, as usual I would say, until you hover over it.